pkg/webui/console/store/reducers/devices.js

```diff
@@ -91,10 +91,7 @@ const devices = (state = defaultState, { type, payload, event }) => {
 
       // Update derived last seen value if possible.
       const derived = {}
-      const lastSeen = getLastSeen(payload)
-      if (lastSeen) {
-        derived.lastSeen = lastSeen
-      }
+      derived.lastSeen = payload.last_seen_at
```

Comment:
Since it is not a derived value anymore, we can remove `lastSeen` completely from the derived store. Just make sure that no other logic is dependent on `lastSeen` anymore (in which case it should use the `last_seen_at` in the device store directly).

Check the `selectDeviceDerivedLastSeen` selector which should be removed and its calls should be replaced with a new selector `selectDeviceLastSeen` which will get the value from the entity directly.

Reply:
I wanted to indeed remove `lastSeen` from the derived state, but then I realized that it is constantly being changed by each event that comes in [here](https://github.com/TheThingsNetwork/lorawan-stack/blob/14ebc66bff863ad1685f6601100b53aa27fdf9b8/pkg/webui/console/store/reducers/devices.js#L208). So if we remove `lastSeen` in this situation, we'd need to update the `last-seen-at` directly in the device, would that be okay - to constantly change a device property?